src/components/MessageCard/MessageCard.test.js

```diff
@@ -61,26 +60,88 @@ describe('Align', () => {
   })
 })
 
-describe('Animation', () => {
-  test('Can customize animationSequence', () => {
-    const wrapper = mount(<MessageCard animationSequence="scale" />)
-    const o = wrapper.find(Animate)
+describe('Visibility', () => {
+  jest.useFakeTimers()
+
+  test('Should be visible by default if there is no image', () => {
+    const onShowSpy = jest.fn()
+    const wrapper = mount(<MessageCard onShow={onShowSpy} />)
+
+    expect(wrapper.state('visible')).toEqual(false)
```

Comment:
It's not anything major and I am okay with pushing it out as it is. However, I don't like accessing the internal state in tests. I would rather check for the prop on a component that we are passing this state to (which is not perfect either, but feels a bit better), than accessing the state directly. Ideally, we could just check for set style on the element, but not sure if this is possible with styled-components and enzyme. What do you think?

Reply:
@jakubjanczyk that's a great point, and I agree that testing that the underlying styled-component has the correct styles would be a better test, since that's what we're trying to accomplish in the end.

There's a jest plugin that would allow us to do this: https://github.com/styled-components/jest-styled-components, which we'll need to install in this repo if we want to go that route. I would leave that up for you and @plbabin to discuss next week!